content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/130-logging.mdx

```diff
@@ -65,6 +65,10 @@ To use event-based logging:
 
 The following example subscribes to all `query` events and write the `duration` and `query` to console:
 
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
```

Comment:
Distance below the tab to the code is surprisingly big:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/183673/157942125-42e2cebe-409c-414f-9378-7358a2a59c08.png">

Reply:
Hm, maybe the TabbedContent and CodeWithResult components don't play together well? (I wasn't sure whether it would work at all...)